components/external-api-framework/org.wso2.carbon.identity.external.api.client/src/main/java/org/wso2/carbon/identity/external/api/client/api/constant/ErrorMessageConstant.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.external.api.client.api.constant;
+
+/**
+ * Error message constants for API Client component.
+ */
+public class ErrorMessageConstant {
+
+    /**
+     * Enum for error messages.
+     */
+    public enum ErrorMessage {
+
+        ERROR_CODE_MISSING_AUTH_TYPE("APICLIENT-65001", "Missing authentication type.",
+                "Authentication type must be provided in the API authentication configuration."),
+        ERROR_CODE_BLANK_AUTH_PROPERTY("APICLIENT-65002", "Blank API authentication property value.",
```

Comment:
```suggestion
        ERROR_CODE_BLANK_AUTH_PROPERTY("APICLIENT-65002", "Missing API authentication property value.",
```

Shall we update this to be consistent with other error messages?

Reply:
Is this for where auth prop value is provide, but it is null or blank. For missing auth prop there is another error `ERROR_CODE_MISSING_AUTH_PROPERTY`